libraries/c_sdk/standard/mqtt/README.md

```diff
@@ -1,18 +1,46 @@
-# MQTT Shim for MQTT V2.x.x APIs
+# MQTT Compatibility Layer for MQTT V2.x.x APIs
 
-This MQTT shim library supports backward compatibility for the MQTT V2.x.x APIs using the [coreMQTT library](../../../coreMQTT/README.md). All the [APIs in MQTT v2.x.x](include/iot_mqtt.h) are supported in this shim implementation. 
-This shim will be supported only as a short term solution to provide backward compatibility. We recommend using the redesigned [coreMQTT library](../../../coreMQTT/README.md) for applications requiring long term support.
+MQTT Compatibility Layer for MQTT V2.x.x APIs, is using the [coreMQTT library](https://github.com/FreeRTOS/coreMQTT/blob/master/README.md). All the [APIs in MQTT v2.x.x](include/iot_mqtt.h) are supported in this implementation. This Compatibility Layer will be supported only as a short term solution to provide backward compatibility. **We recommend using the redesigned [coreMQTT library](https://github.com/FreeRTOS/coreMQTT/blob/master/README.md) for applications requiring long term support.**
 
 
-## MQTT Shim library configs
+## Configs for the Compatibility Layer
 
-Configuration settings of the MQTT shim library. These configurations are available in addition to the configurations of MQTT V2.x.x library.
+Configuration settings for the Compatibility Layer are available in addition to the [configurations of MQTT V2.x.x library](https://docs.aws.amazon.com/freertos/latest/lib-ref/embedded-csdk/v4.0_beta_deprecated/lib-ref/c-sdk/mqtt/mqtt_config.html).
```

Comment:
Please use MQTT Compatibility Layer (instead of just Compatibility Layer)

Reply:
Sure. Thanks